Makefile

```diff
@@ -127,6 +127,10 @@ check_docs:
 
 ## _
 check_format:
+	find apps crates -name '*.rs' \
+	| xargs rustfmt --check \
```

Comment:
Good idea, but it doesn't work because both options are unstable, which I had forgotten about and that is probably why I got into the habit of doing it this way. I think I'll park this idea for now.